JSDemos/Demos/DataGrid/FilteringAPI/description.md

```diff
@@ -1 +1,15 @@
-The **DataGrid** widget ships with an&nbsp;integrated filtering API, allowing you apply filters in&nbsp;code. In&nbsp;this example, the grid&rsquo;s filter function is&nbsp;called to&nbsp;filter data based on&nbsp;selections made within the drop-down menu above the **DataGrid**.
\ No newline at end of file
+**DataGrid** has the following API that you can use to filter data:     
```

Comment:
"to have" is a weak verb and I'd always encourage to find a replacement. I'd probably say "includes".

```suggestion
**DataGrid** includes the following API that you can use to filter data:     
```